subprojects/docs/src/docs/userguide/ear_plugin.adoc

```diff
@@ -92,13 +90,11 @@ The link:{groovyDslPath}/org.gradle.plugins.ear.Ear.html[Ear] class in the API d
 
 Here is an example with the most important customization options:
 
-=== Example: Customization of ear plugin
-
-[source.multi-language-sample,groovy]
-.build.gradle
-----
-include::{samplesPath}/ear/earCustomized/ear/build.gradle[]
-----
+.Customization of ear plugin
+====
+include::sample[dir="ear/earCustomized/groovy",files="ear/build.gradle[]"]
```

Comment:
:x: should be `include::sample[dir="ear/earCustomized/groovy/ear",files="build.gradle[]"]` to keep the displayed file named unchanged

Reply:
OK. Since the layout changed compared to the previous sample (where the war was a subproject of the ear), I was wondering what was best here.